lib/views/screens/upload_screen/pages.dart

```diff
@@ -1,16 +1,139 @@
+import 'dart:io';
+
+import 'package:another_flushbar/flushbar_helper.dart';
+import 'package:deuvox/controller/bloc/uploadimage/upload_image_bloc.dart';
+import 'package:deuvox/data/model/upload_image_model.dart';
 import 'package:deuvox/views/component/common_button.dart';
 import 'package:deuvox/views/component/common_form.dart';
 import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
 
 class UploadScreen extends StatefulWidget {
-  UploadScreen({Key? key}) : super(key: key);
+  const UploadScreen({Key? key}) : super(key: key);
+
 
   @override
   _UploadScreenState createState() => _UploadScreenState();
 }
 
 class _UploadScreenState extends State<UploadScreen> {
   final GlobalKey<FormState> _formKey = GlobalKey<FormState>();
+  UploadImageBloc uploadImageBloc = UploadImageBloc();
+  List<UploadImageModel> uploadImageModels = [];
+  List<File> images = [];
+  final index = ValueNotifier<int>(0);
+
+  Widget displayImage(int index) {
+    if (images.isEmpty) {
+      return SizedBox.shrink();
+    } else {
+      return Image.file(images[index]);
+    }
+  }
+
+  @override
+  void initState() {
+    uploadImageBloc = UploadImageBloc();
+    super.initState();
+  }
+
+  @override
+  void dispose() {
+    uploadImageBloc.close();
+    super.dispose();
+  }
+
+  Future<void> showDialogUploadImage(BuildContext context) {
+    return showDialog(
+        context: context,
+        builder: (context) => BlocProvider<UploadImageBloc>(
+            create: (context) => uploadImageBloc,
+            child: BlocConsumer<UploadImageBloc, UploadImageState>(
+              bloc: uploadImageBloc,
+              listener: (context, state) {
+                if(state is UploadImagePickerSelected) {
+                  images.add(state.image);
+                  UploadImageModel temp = UploadImageModel();
+                  temp.image_name = state.image.path;
+                  uploadImageModels.add(temp);
+                }
+                if(state is UploadImagePickerFailure) {
+                  FlushbarHelper.createError(
+                      message: "Terjadi kesalahan saat memilih image.")
```

Comment:
Jangan gunakan hardcoded-string. Generate variabel constant untuk wording localization. Caranya lihat di sini https://docs.google.com/spreadsheets/d/1be0WGyaMU01uHyYmcogTqgnYwCG0QyDqxrl5QrwVQXE/edit#gid=262867483

Reply:
@Farrize please fix this